examples/nft-royalties/src/test.rs

```diff
@@ -16,15 +16,14 @@ fn test_default_royalty() {
     let e = Env::default();
     let admin = Address::generate(&e);
     let manager = Address::generate(&e);
-    let client = create_client(&e, &admin, &manager);
-
     e.mock_all_auths();
+    let client = create_client(&e, &admin, &manager);
 
     // Mint a token
     let token_id = client.mint(&admin);
 
     // Check royalty info (should use default 10%)
-    let (receiver, amount) = client.get_royalty_info(&token_id, &1000);
+    let (receiver, amount) = client.royalty_info(&token_id, &1000);
```

Comment:
Me:
> `get_royalty_info` is renamed to `royalty_info`, debatable…

Willem:
> It wasn’t renamed because `royalty_info` was part of the trait and thus exposed already. So there were two functions that did the same thing. So I’m open to renaming.